content/03.categorize.md

```diff
@@ -104,6 +105,12 @@ The resulting dataset [@url:https://nihcc.app.box.com/v/ChestXray-NIHCC] consist
 Further, Wang et al. [@arxiv:1705.02315] used this dataset with a unified weakly-supervised multi-label image classification framework, to detect common thoracic diseases.
 It showed superior performance over a benchmark using fully-labeled data.
 
+Another example of semi-automated label generation for hand radiograph segmentation employed positive mining, an iterative procedure that combines manual labeling with automatic processing [@tag:Iglovikov2017_baa].
+First, the initial training set was created by manually labeling 100 of 12,600 unlabeled radiographs that were used to train a model and predict labels for the rest of the dataset.
+Then, through manual inspection, poor quality predictions were discarded, while the initial training set was expanded with the acceptable segmentations, and the process was repeated.
+This procedure had to be repeated 6 times to obtain good quality segmentation labeling for all radiographs, except for a last 100 corner cases that still required manual annotation.
```

Comment:
`6` -> `six`

This reads better if you remove `a last`

Reply:
Done